scripts/server.sh

```diff
@@ -16,6 +16,23 @@ echo adminPassword \'$adminPassword\'
 echo uniqueString \'$uniqueString\'
 echo location \'$location\'
 
+FILE="/var/lib/dpkg/lock-frontend"
+DB="/var/lib/dpkg/lock"
+if [[ -f "$FILE" ]]; then
+  PID=$(lsof -t $FILE)
+  echo "lock-frontend locked by $PID"
+  echo "Killing $PID"
+  kill -9 "${PID##p}"
+  echo "$PID Killed"
+  rm $FILE
+  PID=$(lsof -t $DB)
+  echo "DB locked by $PID"
+  kill -9 "${PID##p}"
+  rm $DB
+  dpkg --configure -a
+fi
```

Comment:
If you desired additional sanity, may want to check that pid was actually killed, as install failure due to zombie proc may not be clear

Reply:
Good call.. I'll get it updated